adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/model/AdapterEvent.java

```diff
@@ -0,0 +1,94 @@
+package org.jvalue.ods.adapterservice.adapter.model;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.Serializable;
+
+/**
+ * This Event will be sent to the transformation service after data import execution.
+ */
+public class AdapterEvent implements Serializable {
+    private String data;
+    private String dataLocation;
+    private long datasourceId;
+
+  /**
+   * Default constructor
+   * @param datasourceId Adapter config identifier (referenced by pipeline config)
+   * @param data imported data (by adapter import)
+   * @param dataLocation location of the data on adapter service (--> endpoint)
+   */
+    public AdapterEvent(long datasourceId, String data, String dataLocation) {
+      this.datasourceId = datasourceId;
+      this.data = data;
```

Comment:
please use a thin event first (i.e. do not include the data itself). This reduces the number of changes necessary to make this pull request work. Afterward, we can change that, but in a different pull request.

Reply:
Currently the system is capable to handle both. 
If `data` exists (!=null) within the event it will use that. Else it will get the data from the datasource link `dataLocation`  (also in the event).